phabricatoremails/settings.py

```diff
@@ -120,15 +123,18 @@ def mail(self):
         return _parse_mail(self._config, self.logger)
 
     @classmethod
-    def load(cls):
-        """Load and parse settings from a settings.ini file in the package directory."""
+    def load(cls, environment_path=None):
```

Comment:
I would suggest renaming this to `settings_path`, given that it is technically agnostic to where the settings path is coming from (as understood from the docstring).